core/storage/user/gae_models_test.py

```diff
@@ -170,3 +170,75 @@ def test_instance_stores_correct_data(self):
             query_model.edited_at_least_n_exps, edited_at_least_n_exps)
         self.assertEqual(
             query_model.edited_fewer_than_n_exps, edited_fewer_than_n_exps)
+
+
+class UserSkillMasteryModelTests(test_utils.GenericTestBase):
+    """Tests for UserSkillMasteryModel."""
+
+    USER_ID = 'user_id'
+    SKILL_ID_1 = 'skill_id_1'
+    SKILL_ID_2 = 'skill_id_2'
+    DEGREE_OF_MASTERY = 0.5
+
+    def setUp(self):
+        super(UserSkillMasteryModelTests, self).setUp()
+        user_models.UserSkillMasteryModel(
+            id=user_models.UserSkillMasteryModel.construct_model_id(
+                self.USER_ID, self.SKILL_ID_1),
+            user_id=self.USER_ID,
+            skill_id=self.SKILL_ID_1,
+            degree_of_mastery=self.DEGREE_OF_MASTERY).put()
+
+        user_models.UserSkillMasteryModel(
+            id=user_models.UserSkillMasteryModel.construct_model_id(
+                self.USER_ID, self.SKILL_ID_2),
+            user_id=self.USER_ID,
+            skill_id=self.SKILL_ID_2,
+            degree_of_mastery=self.DEGREE_OF_MASTERY).put()
+
+    def test_construct_model_id(self):
+        constructed_model_id = (
+            user_models.UserSkillMasteryModel.construct_model_id(
+                self.USER_ID, self.SKILL_ID_1))
+
+        self.assertEqual(constructed_model_id, 'user_id.skill_id_1')
+
+    def test_get_success(self):
+        constructed_model_id = (
+            user_models.UserSkillMasteryModel.construct_model_id(
+                self.USER_ID, self.SKILL_ID_1))
+        retrieved_object = user_models.UserSkillMasteryModel.get(
+            constructed_model_id)
+
+        self.assertEqual(retrieved_object.user_id, 'user_id')
+        self.assertEqual(retrieved_object.skill_id, 'skill_id_1')
+        self.assertEqual(retrieved_object.degree_of_mastery, 0.5)
+
+    def test_get_failure(self):
+        retrieved_object = user_models.UserSkillMasteryModel.get(
+            'unknown_model_id', False)
```

Comment:
Change to strict=False. If there's a default arg then you should specify the arg name as well when calling.

@apb7 @kevinlee12 -- we should be catching this in the linter?

Reply:
Hi @seanlip, the ExplicitKwargsChecker does not work for functions imported from another module, at the moment. Please refer to Issue #5004 for further details. Thanks!

Reply:
Oh yes, I forgot that. Thanks!

@WickedBrat if you can fix this last thing, it's fine to go!